canonical/charmhub.io
# Make beta the main app

## Done
- refactor the main app to use store-base
- remove beta app
## How to QA
- Go to https://charmhub-io-1599.demos.haus/store.json and ensure it returns a list of packages with the total page number
- Go to https://charmhub-io-1599.demos.haus/beta-store and ensure it displays the UI with package cards
- Go to https://charmhub-io-1599.demos.haus/account.json and ensure it returns the authenticated user's details
## Note
- This is a breaking change that should not be merged into main until the frontend has been implemented and thoroughly tested.
- The login route presently returns a json containing the token (stored in session) for the authenticated user.
## Issue / Card https://warthogs.atlassian.net/browse/WD-4228
Fixes #

## Screenshots
